cluster-sync/okd-4.1/provider.sh

```diff
@@ -7,7 +7,15 @@ function seed_images(){
 }
 
 function configure_local_storage() {
-  #Set the default storage class.
+  retry_counter=0
+  while [[ $retry_counter -lt 60 ]] && [ "$sc" != "local" ]; do
```

Comment:
sc not defined

Reply:
Right, so $sc will not match "local" and it gets defined in the loop.

Reply:
I guess I can save us 5 seconds if I copy
```bash
sc=`_kubectl get sc local -o=jsonpath="{.metadata.name}"`
```
outside of the loop, if it exists already, the loop doesn't get entered.

Reply:
Yeah nbd because not using "set -u"

Reply:
I likely don't need retry_counter=0 either, but I wasn't sure what would happen if I added one to an undefined value. If it would figure out hey it should be a number and thus I will initalize it to 0 and add 1.